src/exercises/day-1/luhn.rs

```diff
@@ -15,25 +15,30 @@
 // ANCHOR: solution
 // ANCHOR: luhn
 pub fn luhn(cc_number: &str) -> bool {
-    // ANCHOR_END: luhn
```

Comment:
Please don't remove the comment, it's necessary to avoid showing the solution on [the exercise page](https://google.github.io/comprehensive-rust/exercises/day-1/luhn.html) :sweat_smile: